src/script/cpp_api/s_sscsm.h

```diff
@@ -0,0 +1,15 @@
+// SPDX-FileCopyrightText: 2025 Luanti authors
+//
+// SPDX-License-Identifier: LGPL-2.1-or-later
+
+#pragma once
+
+#include "cpp_api/s_base.h"
+
+class ScriptApiSSCSM : virtual public ScriptApiBase
+{
+public:
+	void load_mods(const std::vector<std::pair<std::string, std::string>> &mods);
```

Comment:
with the "set of named scripts + `require`" approach, this simplifies to something like `load_scripts(const std::vector<std::string> &scripts)`.